rune/libenclave/runelet.go

```diff
@@ -68,8 +68,16 @@ func StartInitialization(cmd []string, cfg *RuneletConfig) (exitCode int32, err
 		}
 
 		// Launch a remote attestation to the enclave runtime.
-		if err = rt.LaunchAttestation(); err != nil {
-			return 1, err
+		product := "development"
+		quote_type := "SGX_UNLINKABLE_SIGNATURE"
+		if config.RaEpidQuoteType == "linkable" {
```

Comment:
Please change config.RaEpidQuoteType to integer and use QuoteSignatureTypeUnlinkable or QuoteSignatureTypeLinkable defined in intelsgx package.

Reply:
> Please change config.RaEpidQuoteType to integer and use QuoteSignatureTypeUnlinkable or QuoteSignatureTypeLinkable defined in intelsgx package.

 The `config.RaEpidQuoteType` located in `Spec.annotations`， the type of `Spec.annotations` is `map[string]string`. If given a integer value to `config.RaEpidQuoteType`, there would be an wrong "cannot unmarshal number into Go struct field Spec.annotations of type string"

Reply:
> > Please change config.RaEpidQuoteType to integer and use QuoteSignatureTypeUnlinkable or QuoteSignatureTypeLinkable defined in intelsgx package.
> 
> The `config.RaEpidQuoteType` located in `Spec.annotations`， the type of `Spec.annotations` is `map[string]string`. If given a `integer` value to `config.RaEpidQuoteType`, there would be an wrong "cannot unmarshal number into Go struct field Spec.annotations of type string"

If we use a function like `strconv.ParseUint` to transfer the `config.RaEpidQuoteType` to a `integer`, the value of a field of  `Spec.annotations` which isn't a digital string always returns 0 using a function like `strconv.ParseUint`.